srcpkgs/tpm2-tss/template

```diff
@@ -24,6 +27,8 @@ pre_build() {
 }
 
 post_install() {
+	rm -rf ${DESTDIR}/etc/tmpfiles.d
+	rm -rf ${DESTDIR}/etc/sysusers.d
```

Comment:
install can only run once, otherwise there will be conflicts, so we should use `rm -r` to make sure this errors out when it's no longer required